l3xz_mapping/Startpoint startpoint
---
bool accepted
